deck.md

```diff
@@ -1366,6 +1366,125 @@ export default async function users(fastify) {
 
 ---
 
+class: branded
+
+# Step 13: Exercise 💻
+
+- Let's create an Fastify application using **Typescript**.
+
+- We will transpose the application that you did in the [Step 09](#step-9-decorators) to Typescript.
+
+- Use `declaration merging` to add the custom decorator (authenticate) property to `FastifyInstance`
+
+- Use `typebox` to transform JSON Schema into types
+
+---
+
+# Step 13: Solution/1
+
+```ts
+// routes/login.ts
+import { Type, Static } from '@sinclair/typebox'
+import { FastifyInstance, FastifyRequest } from 'fastify'
+
+const BodySchema = Type.Object({
+  username: Type.String(),
+  password: Type.String(),
+})
+
+// Generate type from JSON Schema
+type BodySchema = Static<typeof BodySchema>
+
+const ResponseSchema = Type.Object({
+  username: Type.String(),
+  password: Type.String(),
+})
+
+type ResponseSchema = Static<typeof ResponseSchema>
+
+const schema = {
+  body: BodySchema,
+  response: {
+    200: ResponseSchema
+  }
+}
```

Comment:
This slide does not fit completely on the screen.

I think that removing a couple of newlines would fix the issue.

Reply:
What's the resolution of your screen?

Reply:
I've tested various resoultions and the slides seems to be designed to give the same result at any resolution.

I think the problem is that I'm using Firefox which seems to render the example differently from Chrome taking more space.

Reply:
I have removed some lines, could you check again?

Reply:
It's perfect now!